bot/exts/evergreen/githubinfo.py

```diff
@@ -0,0 +1,95 @@
+import logging
+from datetime import datetime
+from typing import Optional
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+log = logging.getLogger(__name__)
+
+
+class GithubInfo(commands.Cog):
+    """Fetches info from GitHub."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @staticmethod
+    async def fetch_data(url: str) -> dict:
+        """Retrieve data as a dictionary."""
+        async with aiohttp.ClientSession() as cs:
+            async with cs.get(url) as r:
+                return await r.json()
+
+    @commands.command(name='github', aliases=['gh'])
+    @commands.cooldown(1, 5, BucketType.user)
+    async def get_github_info(self, ctx: commands.Context, username: Optional[str]) -> None:
+        """
+        Fetches a user's GitHub information.
+
+        Username is optional and sends the help command if not specified.
+
+        Usage: .gh [username]
+        """
+        if username is None:
+            await ctx.invoke(self.bot.get_command('help'), 'github')
+            return
+
+        async with ctx.typing():
+
+            user_data = await self.fetch_data(f"https://api.github.com/users/{username}")
+
+            # User_data will not have a message key if the user exists
+            if user_data.get('message') is not None:
+                await ctx.send(embed=discord.Embed(title=f"The profile for `{username}` was not found.",
+                                                   colour=0xff0022))
```

Comment:
```suggestion
                                                   colour=discord.Colour.red()))
```
This default color is generally used for error messages from SeasonalBot. You could also do `from discord import Colour`, so you'll be able to do `Colour.red()`, instead of having to include `discord.` every time.